pom.xml

```diff
@@ -59,7 +59,7 @@
         <dependency>
             <groupId>ru.yandex.clickhouse</groupId>
             <artifactId>clickhouse-jdbc</artifactId>
-            <version>0.1.40</version>
```

Comment:
Round question: does it make sense this dependency explicitly declared here? Why not just inherit from sirius-db?

Reply:
sirius-db only declares as a test dependency, as sirius-db has no clickhouse specific code. biz does, hence the dependency here...